proto/osmosis/incentives/autostaking.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package osmosis.incentives;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/x/incentives/types";
+
+message AutoStaking {
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer)         = true;
+
+  string address = 1 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+  string autostaking_validator = 2;
+  string autostaking_rate = 3 [
```

Comment:
Can we implement this without an autostaking rate for the first version, and just use the governance param?

I think this is going to make many types of batch optimizations that we will likely need to later do for speed harder, and we should consider this once we know where stand in speed.

Reply:
Disabled